doc/rbd/rbd-nomad.rst

```diff
@@ -73,38 +73,54 @@ Configure ceph-csi
 Setup Ceph Client Authentication
 --------------------------------
 
-Create a new user for nomad and `ceph-csi`. Execute the following and
-record the generated key::
+Create a new user for nomad and `ceph-csi`. Execute the following command and
+record the generated key:
 
-    $ ceph auth get-or-create client.nomad mon 'profile rbd' osd 'profile rbd pool=nomad' mgr 'profile rbd pool=nomad'
+.. code-block:: console
+
+  $ ceph auth get-or-create client.nomad mon 'profile rbd' osd 'profile rbd pool=nomad' mgr 'profile rbd pool=nomad'
     [client.nomad]
         key = AQAlh9Rgg2vrDxAARy25T7KHabs6iskSHpAEAQ==
 
 
 Configure Nomad  
 ---------------
 
-By default Nomad doesn't allow containers to use privileged mode.
-Edit the nomad configuration file by adding this configuration block to `/etc/nomad.d/nomad.hcl`::
+Configuring Nomad to Allow Containers to Use Privileged Mode
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default, Nomad doesn't allow containers to use privileged mode. We must
+configure Nomad so that it allows containers to use privileged mode. Edit the
+Nomad configuration file by adding the following configuration block to
+`/etc/nomad.d/nomad.hcl`::
 
     plugin "docker" {
         config {
         allow_privileged = true
         }
     }
 
+Loading the rbd module
+~~~~~~~~~~~~~~~~~~~~~~
+
+Nomad must have the `rbd` module loaded. Run the following command to confirm that the `rbd` module is loaded:
+
+.. code-block:: console
+
+  $ lsmod |grep rbd
+  rbd                    94208  2
+  libceph               364544  1 rbd
 
-Nomad must have `rbd` module loaded, check if it's the case.::
+If the `rbd` module is not loaded, load it:
 
-        $ lsmod |grep rbd
-        rbd                    94208  2
-        libceph               364544  1 rbd
+.. prompt:: bash $
```

Comment:
Looking at the preview (https://ceph--42806.org.readthedocs.build/en/42806/rbd/rbd-nomad/), using `code-block:: console` for commands with output and `prompt:: bash $` for commands without output leads to `$` being highlighted in the `code-block` case but not in the `prompt` case.  I think command snippets should look the same regardless of whether output is present so we probably need to pick one and stick to it for consistency.

Reply:
I agree with you, but there's a technical problem (as far as I know).

If you use "prompt", then the reader can copy and paste the entire command and the prompt will be left out of the string copied to the clipboard. This is great, and this is the desired behavior.

However, you cannot print a prompt line and console output in an unbroken way if you use the "prompt" directive. This is why I have used code-block.

In the past, I have used the prompt directive and followed the prompt with a line that says something like this:

     You should see output like the following:

If you think that's a more natural solution, I'll start doing that. I just need to know what readers of this stuff think.

Reply:
I think separating the output from the command and effectively making it a separate paragraph would be too verbose, especially given that the output is usually just a few lines.

What you did with `prompt` followed by `code-block` seems to work in that `$` doesn't make it into the clipboard but the indentation is still messed up.  For example I see

```
$ ceph auth get-or-create client.nomad mon 'profile rbd' osd 'profile rbd pool=nomad' mgr 'profile rbd pool=nomad'
  [client.nomad]
      key = AQAlh9Rgg2vrDxAARy25T7KHabs6iskSHpAEAQ==
```
in the preview but it should look like

```
$ ceph auth get-or-create client.nomad mon 'profile rbd' osd 'profile rbd pool=nomad' mgr 'profile rbd pool=nomad'
[client.nomad]
        key = AQAlh9Rgg2vrDxAARy25T7KHabs6iskSHpAEAQ==
```

Reply:
Sorry, apparently I looked at a weird mix of the old and the new previews.  `prompt` followed by `code-block` doesn't work.  Here is what I see at https://ceph--42806.org.readthedocs.build/en/42806/rbd/rbd-nomad/ after restarting the browser:

![prompt](https://user-images.githubusercontent.com/567341/130609764-55dcc5f4-bba6-4ee9-ab10-cb3d03439879.jpg)

and

![prompt](https://user-images.githubusercontent.com/567341/130609530-d175c7e2-3209-4a5b-a91f-46ac16c5435e.jpg)

Honestly, I would punt on trying to keep the prompt out of the clipboard and just use `code-block` (or even raw text as it is now) for consistency and succinctness.  But I'm not a docs person, so up to you ;)

Reply:
I looked at other parts of documentation and we seem to be mixing `prompt` and `code-block` there so it is not worth fussing about.  I pushed an update mostly reverting to what you had before.  Sorry for the noise!